setup/db/db/schema-490to491.sql

```diff
@@ -23,3 +23,11 @@
 UPDATE `cloud`.`roles` SET `description`='Default user role' WHERE `id`=4 AND `role_type`='User' AND `description`='Default Root Admin role';
 
 
+ALTER TABLE cloud.load_balancer_cert_map ENGINE=INNODB;
+ALTER TABLE cloud.monitoring_services ENGINE=INNODB;
+ALTER TABLE cloud.nic_ip_alias ENGINE=INNODB;
+ALTER TABLE cloud.sslcerts ENGINE=INNODB;
+ALTER TABLE cloud.op_lock ENGINE=INNODB;
+ALTER TABLE cloud.op_nwgrp_work ENGINE=INNODB;
```

Comment:
Yes, they have existed since 4.0. I checked git history, I think there is nothing specific to this table, they can be migrated to InnoDB. I'm running a local 4.9.0 CloudStack environment with these changes and it is working alright.